README.md

```diff
@@ -28,14 +28,34 @@ Booking App is a full-stack web application developed as a final project for the
 
 - User Registration and Login: Users can create accounts and log in to the app.
 - Place Listing: Users can upload their own places for rental. Add images either via link or direct upload.
-- User Dashboard: Logged-in users can manage their listings and view their rental history.
+- User Dashboard: Logged-in users can manage their listings and view their bookings.
 
 ## Installation
 
 ### Frontend
 
 ```bash
-cd frontend
+cd client
 npm install
 npm start
+
+### Backend
+cd api
+npm install
```

Comment:
Maybe we should add nodemon (or it's in it?) installation, because I don't know how it will work with npm. I think then you have to restart server every time after you send something to be.

Reply:
Very good idea, I will add installing nodemon.